internal/pkg/describe/service.go

```diff
@@ -294,17 +318,46 @@ func (d *appRunnerServiceDescriber) Service() (*apprunner.Service, error) {
 	return service, nil
 }
 
+// IsPrivate returns true if the service is configured as non-public.
+func (d *appRunnerServiceDescriber) IsPrivate() (bool, error) {
+	_, err := d.vpcIngressConnectionARN()
+	if err != nil {
+		var errNotFound *errVPCIngressConnectionNotFound
+		if errors.As(err, &errNotFound) {
```

Comment:
nit: with the sentinel error we can use `if errors.Is(err, errVPCIngressConnectionNotFound)`  (same in the `ServiceURL` method)